src/engine.rs

```diff
@@ -110,14 +103,15 @@ impl Engine {
             ..
         } = set;
 
-        let blocker_options = BlockerOptions {
-            enable_optimizations: optimize,
-        };
+        let memory = FlatBufferBuilder::make_flatbuffer(network_filters, optimize);
+
+        let filter_data_context = FilterDataContext::new(memory);
 
         Self {
-            blocker: Blocker::new(network_filters, &blocker_options),
+            blocker: Blocker::from_context(filter_data_context.clone()),
```

Comment:
this `.clone()` should be `FilterDataContextRef::clone`

Reply:
done